Project/Source Files/Screens/Recipies/RecipiesTableViewController.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  RecipiesViewController.swift
+//  NetguruCollegeApp
+//
+
+import UIKit
+
+class RecipiesTableViewController: UIViewController {
+
+    @IBOutlet var tableView: UITableView!
+    var dataBase = [
+        Recipe(image: UIImage(named: "TestingImage")!,
+               title: "Let's cook something",
+               ingredients: "Garlic, beef, salt, Garlic, beef, salt, Garlic, beef, salt"),
+        Recipe(image: UIImage(named: "TestingImage2")!,
+               title: "Another cool recipe",
+               ingredients: "Mango, Melon, Water")
+    ]
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupView()
+    }
+    private func setupView() {
+        tableView.register(UINib(nibName: "RecipiesTableViewCell", bundle: nil),
```

Comment:
What would be also cool: adding a 
```
class var identifier: String {
        return "RecipiesTableViewCell"
    }
````
to the table view cell class and using it :)